android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMPBrowserActivity.java

```diff
@@ -61,13 +63,20 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
 
       @Override
       public boolean shouldOverrideUrlLoading(WebView view, String url) {
-        if (!url.toLowerCase().equals("about:blank")) {
-          if (url.startsWith("keyman:download")) {
+        String lowerURL = url.toLowerCase();
```

Comment:
I was lower-casing to avoid issues with mixed-case BCP 47 tags.

I can pass the original URL to the handler..

Reply:
> I was lower-casing to avoid issues with mixed-case BCP 47 tags.

That needs to be done just at bcp 47 handling then. I think we probably need to have a BCP47 equivalence function defined for consistent use everywhere.

Reply:
For now, I'll lower-case the ID in JSONUtils when checking for matching language ID.

> I think we probably need to have a BCP47 equivalence function defined for consistent use everywhere.

I'll split that into a separate issue